_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -4,14 +4,27 @@ title: Server-Side Rendering
 
 In Spartacus, server-side rendering allows you to render static versions of pages on the server side. This speeds up response times, assists with SEO, and allows the application to render more quickly. After Angular has bootstrapped, users of your site will have the full experience.
 
-***
+---
 
 **Table of Contents**
 
 - This will become a table of contents (this text will be scrapped).
-{:toc}
+  {:toc}
+
+---
+
+## Which pages to server-side render?
+
+It is generally recommended to server-side render pages which don't change very frequently and don't contain a notion of personalization. In the e-commerce world, this usually means that you shouldn't SSR the following pages:
```

Comment:
- Pages that will be crawled by site indexers or bots for search engine indexing and SEO purposes
- Pages with static content which doesn't change often
- Pages that will not contain personalized content

Reason for this is that we recommend to cache most of these to avoid overloading the SSR servers

Reply:
Thanks, changed.